### PR TITLE
SDK 1.3.1 release notes

### PR DESCRIPTION
**Pull request by keharper**
## Purpose of this pull request

This pull request (PR) creates the release notes for Admin UI SDK 1.3.1.

## Affected pages

- ...

## Links to Magento Open Source code

- ...

### CODE REVIEW

File: src/pages/admin-ui-sdk/extension-points/order/mass-action.md [hunk 1]
```diff
@@ -50,6 +51,7 @@ order: {
 | `confirm.message` | string | No | The message displayed on the confirmation dialog for a mass action. |
 | `confirm.title` | string | No | The title of a dialog that confirms the mass action. |
 | `label` | string | Yes | An Action label to display in the Mass Actions grid. |
+| `title` | string | No | An optional page title for the action. If not specified, the label is used.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/order/mass-action.md:54]:
```suggestion
| `title` | string | No | An optional page title for the action. If not specified, the label is used. |
```

File: src/pages/admin-ui-sdk/extension-points/product/mass-action.md [hunk 2]
```diff
@@ -50,6 +51,7 @@ product: {
 | `confirm.message` | string | No | The message displayed on the confirmation dialog for a mass action |
 | `confirm.title` | string | No | The title of a dialog that confirms the mass action |
 | `label` | string | Yes | An Action label to display in the Mass Actions grid |
+| `title` | string | No | An optional page title for the action. If not specified, the label is used.
```
Comment (asalloum5) [inline src/pages/admin-ui-sdk/extension-points/product/mass-action.md:54]:
```suggestion
| `title` | string | No | An optional page title for the action. If not specified, the label is used. |
```

**Review by asalloum5 — APPROVED** [review]

File: src/pages/admin-ui-sdk/release-notes.md [hunk 3]
```diff
@@ -8,6 +8,26 @@ keywords:
...
+
+* Added the ability to optionally specify a page title for a mass action. <!-- CEXT-2598 -->
+
+* The Admin UI SDK cache is now flushed weekly instead of every 12 hours. <!-- CEXT-2643 -->
+
+* The Admin UI SDK now supports PHP 8.3. <!-- CEXT-2724 -->
+
+### Bug fixes
+
+* The Admin now displays extensions correctly in the Admin when they are loaded from the extension manager registry. <!-- CEXT-2595 -->
```
Comment (jhadobe) [inline src/pages/admin-ui-sdk/release-notes.md:29]: Maybe reword so that "the admin" isn't repeated

**Review by jhadobe — APPROVED** [review]
